overwrites/w3c.json

```diff
@@ -20,6 +20,13 @@
     "encoding": {
         "delete": true
     },
+    "hr-time": {
+        "renameTo": "hr-time-1",
+        "aliasOf": { "replaceWith": "hr-time-2" }
+    },
+    "hr-time-1": {
+        "href": { "replaceWith": "https://www.w3.org/TR/hr-time-1/" }
+    },
```

Comment:
To clarify, here, we're trying to do:

1. Rename `hr-time` to `hr-time-1`
2. alias `hr-time` to `hr-time-2`
3. repace `hr-time-1` `href` with `https://www.w3.org/TR/hr-time-1/`

The way we're going about this seems a bit brittle, though, and not super clear.

I think this is essentially due to the fact that I chose an object to express what should essentially be an array of actions. Consider the following:

```json
[
  { "id": "hr-time", "action": "renameTo", "newId": "hr-time-1" },
  { "id": "hr-time", "action": "createAlias", "aliasOf": "hr-time-2" },
  { "id": "hr-time-1", "action": "replaceProps", "prop": "href", "value": "https://www.w3.org/TR/hr-time-1/" } 
]
```

This is uglier, I agree, but feels a lot less like it's relying on magic, no?

Reply:
I&nbsp;feel&nbsp;like that&nbsp;should be&nbsp;its&nbsp;own&nbsp;PR.

Reply:
Fair enough.